bin/get-firefox

```diff
@@ -28,160 +29,162 @@ const index = require('..'),
+                }
+            },
+            {
+                title: "Verify checksum",
+                enabled: () => cli.flags.check,
+                skip: (context) => context.noChecksums || !context.checksums,
+                task: async (context) => {
+                    context.stream = index.check(context.buffer, context.name, context.checksums);
+                    context.buffer = await streamToPromise(context.stream);
+                }
+            },
+            {
+                title: "Extract",
+                enabled: () => cli.flags.extract,
+                task: (context, task) => {
+                    task.output = `Extracting the archive to ${chalk.blue(cli.flags.target || "./")}`;
+                    return index.extract(context.buffer, cli.flags.target);
+                }
+            },
+            {
+                title: "Write file to disk",
+                enabled: () => !cli.flags.extract,
+                task: (context, task) => {
+                    const saveAs = cli.flags.target || context.name;
+                    task.output = `Saving as ${chalk.blue(saveAs)}`;
+                    return index.writeFile(saveAs, context.buffer);
+                }
             }
-        },
-        {
-            title: "Download",
-            task: async (context, task) => {
-                task.output = `Downloading from ${chalk.blue(context.url)}`;
-                context.stream = await index.downloadFirefox(context.container);
-                context.buffer = await streamToPromise(context.stream);
-            }
-        },
-        {
-            title: "Verify checksum",
-            enabled: () => cli.flags.check,
-            skip: (context) => context.noChecksums || !context.checksums,
-            task: async (context) => {
-                context.stream = index.check(context.buffer, context.name, context.checksums);
-                context.buffer = await streamToPromise(context.stream);
-            }
-        },
-        {
-            title: "Extract",
-            enabled: () => cli.flags.extract,
-            task: (context, task) => {
-                task.output = `Extracting the archive to ${chalk.blue(cli.flags.target || "./")}`;
-                return index.extract(context.buffer, cli.flags.target);
-            }
-        },
-        {
-            title: "Write file to disk",
-            enabled: () => !cli.flags.extract,
-            task: (context, task) => {
-                const saveAs = cli.flags.target || context.name;
-                task.output = `Saving as ${chalk.blue(saveAs)}`;
-                return index.writeFile(saveAs, context.buffer);
-            }
-        }
-    ], {
-        renderer: cli.flags.verbose ? 'verbose' : 'default'
-    }).run();
-}
+        ], {
+            renderer: cli.flags.verbose ? 'verbose' : 'default'
+        }).run();
+    }
+}).catch((error)=> console.log(error))
```

Comment:
Catching the error like this would mean that the binary doesn't exit with an error exit code.

Reply:
Yes, this is testing code, I didn't mean to open a PR in this repo 🙂